cogs/sendRandomPost.py

```diff
@@ -1,26 +1,24 @@
 import discord
-from discord import app_commands
 from discord.ext import commands
 
 from api import getRandomMarketingPost
 import io
 import aiohttp
 
 
-class sendRandomPostCog(commands.Cog):
-
+class sendRandomPostCog(commands.Cog, name="Send random post commands"):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    @app_commands.command(name="random-willow-motivation", description="Send a random motivational post!")
+    @commands.command(name="random-willow-motivation", help="Send a random motivational post!")
```

Comment:
When fixing the syncing issue, double check that the descriptions still appear properly in the UI when typing '/' as it's been removed from the parameters.